app/javascript/scimo/styles/Diffusion.css

```diff
@@ -0,0 +1,208 @@
+/* ==========================================================================
+   HTML5 display definitions
+   ========================================================================== */
+
+/**
+ * Correct `block` display not defined in IE 6/7/8/9 and Firefox 3.
+ */
+
+
+ /*body*/
+ /*{	margin:				0;*/
+ /*    padding:			0;*/
+ /*}*/
+
+
+ article,
+ aside,
+ details,
+ figcaption,
+ figure,
+ footer,
+ header,
+ hgroup,
+ main,
+ nav,
+ section,
+ summary {
+     display: block;
+ }
+
+ /**
+  * Correct `inline-block` display not defined in IE 6/7/8/9 and Firefox 3.
+  */
+
+ audio,
+ canvas,
+ video {
+     display: inline-block;
+     *display: inline;
+     *zoom: 1;
+ }
+
+ /**
+  * Prevent modern browsers from displaying `audio` without controls.
+  * Remove excess height in iOS 5 devices.
+  */
+
+ audio:not([controls]) {
+     display: none;
+     height: 0;
+ }
+
+ /**
+  * Address styling not present in IE 7/8/9, Firefox 3, and Safari 4.
+  * Known issue: no IE 6 support.
+  */
+
+ [hidden] {
+     display: none;
+ }
+
+ section#top {
```

Comment:
I think that this css file is not implementing any styles because the react file does not have a section with id = "top" but if you changed the class names to reflect the id="diffusion" that you put in the react file, the styles should be picked up and won't cause unintended changes elsewhere in the app!